Resources/Prototypes/Entities/Objects/Devices/Circuitboards/Machine/production.yml

```diff
@@ -640,6 +640,25 @@
           amount: 4
           defaultPrototype: PowerCellSmall
 
+- type: entity
+  id: SMESAdvancedMachineCircuitboard
+  parent: BaseMachineCircuitboard
+  name: advanced SMES machine board
+  description: A machine printed circuit board for an Advanced SMES.
+  components:
+  - type: Sprite
+    sprite: Objects/Misc/module.rsi
+    state: power_mod
+  - type: MachineBoard
+    prototype: SMESAdvancedEmpty
+    stackRequirements:
+      Capacitor: 2
+      CableHV: 20
+    componentRequirements:
+      PowerCell:
+        amount: 4
+        defaultPrototype: PowerCellSmall
```

Comment:
Thoughts on maybe making it use 4 `PowerCellMedium`s?

Reply:
I did, but I was suggested to make it slightly cheaper, so I went with smalls. I was thinking of changing it back.

Reply:
It is double the capacity so I would at least use 6 smalls so that it isn't equivalent.

Reply:
Hm, true. I'll use 4 mediums as science can usually print those like candy by the time this is researched.